cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/validator/PreCompileValidatorImpl.java

```diff
@@ -203,6 +206,38 @@ public void validateResultsSection(
         }
     }
 
+    @Override
+    public void validateNoDuplicateInputs(List<Input> inputs, Input element) {
```

Comment:
compiler complains because of generic collections (raw  collections), I can't call that method directly with List<SpecificType> + there are minor diffs like the error message - I think it's ok for now